### ISSUE TITLE
v1.12.0: TypeError: Cannot read property 'location' of null

### ISSUE THREAD

**Issue by JClackett** [dotansimha/graphql-code-generator#3407]
**Describe the bug**
running "graphql-codegen" now gives the error

```
TypeError: Cannot read property 'location' of null
        at /Users/<me>/Documents/Apps/<project>/node_modules/@graphql-toolkit/core/index.cjs.js:622:90
```

**To Reproduce**

upgrading to 1.12.0

3. My `codegen.yml` config file:

```yml
schema: http://localhost:5000/graphql
documents: ./src/**/*.{ts,tsx}
overwrite: true
generates:
  ./src/graphql/types.tsx:
    config:
      withHooks: true
      withComponent: false
      withHOC: false
      withMutationFn: false
      addDocBlocks: false
    plugins:
      - typescript
      - typescript-operations
      - typescript-react-apollo

```

**Expected behavior**
generates types without failing

**Environment:**

- OS: MacOS
- `@graphql-codegen/1.12.0`: 
- NodeJS: 

I changed nothing in my codebase but upgrading now creates this error

**Comment by ardatan** [comment]
Could you try with the canary version?
`1.12.1-alpha-ca51a54f.1+ca51a54f`

**Comment by JClackett** [comment]
@ardatan working!

**Comment by ardatan** [comment]
Let's keep it open until we release a patch version :)

**Comment by JClackett** [comment]
Oops my bad - tabbed to the close and comment button by accident ;) Thanks for the fix

**Comment by dotansimha** [comment]
Fixed in `v1.12.1` :) 
Thanks @ardatan
